src/components/NavLinks/NavLinks.component.tsx

```
@@ -0,0 +1,57 @@
+                                    <ExpandLessIcon />
+                                ) : (
+                                    <ExpandMoreIcon />
+                                ))}
+                            {navItem.count && (
+                                <StyledBadge badgeContent={navItem.count} />
+                            )}
+                        </StyledListItemButton>
+                    </ListItem>
+                    {navItem.child && (
+                        <Collapse in={open === navItem.id}>
+                            <NavLinks navItems={navItem.child}></NavLinks>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)